config/kcp/apiexport_release.yaml

```diff
@@ -24,6 +24,9 @@ spec:
   - resource: "pipelineruns"
     group: "tekton.dev"
     identityHash: pipeline-service
+  - resource: "enterprisecontract"
```

Comment:
wrong name - https://github.com/hacbs-contract/enterprise-contract-controller/blob/main/config/crd/bases/appstudio.redhat.com_enterprisecontractpolicies.yaml#L14

Reply:
You are too fast :D. I was going through it now and found the name and the id hash i added were wrong. Please take a look again.